tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -296,28 +299,38 @@ end
 
 # A handy helper function to rescale our dataset.
 function standardize(x)
-    return (x .- mean(x; dims=1)) ./ std(x; dims=1), x
+    return (x .- mean(Array(x); dims=1)) ./ std(Array(x); dims=1)
+end
+
+function standardize(x, orig)
+    return (x .- mean(Array(orig); dims=1)) ./ std(Array(orig); dims=1)
 end
 
 # Another helper function to unstandardize our datasets.
 function unstandardize(x, orig)
-    return (x .+ mean(orig; dims=1)) .* std(orig; dims=1)
+    return x .* std(Array(orig); dims=1) .+ mean(Array(orig); dims=1)
+end
+
+function unstandardize(x, mean_train, std_train)
+    return x .* std_train .+ mean_train
```

Comment:
Yes, in this particular context, `orig` is a `Matrix` type. I added the Array coercion just in case - too safety aware like writing a Python def to reduce test efforts.
I shall remove the `Array` coercion then, as it's not written as a generic function but for this particular use.